x-pack/plugins/security_solution/server/request_context_factory.ts

```diff
@@ -116,10 +115,7 @@ export class RequestContextFactory implements IRequestContextFactory {
         }
 
         const username = security?.authc.getCurrentUser(request)?.username || 'elastic';
-        return new ExceptionListClient({
-          savedObjectsClient: context.core.savedObjects.client,
-          user: username,
-        });
+        return lists.getExceptionListClient(context.core.savedObjects.client, username);
```

Comment:
FYI - I'm going to search for other usages of `ExceptionListClient` directly and instead see if they can be changed to invoke the `getExceptionListClient()` factory from the `lists` plugin